definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -65,6 +66,9 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "LinkSealedClaimDocument", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "LinkDefendant", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "TestSupportUpdate", "UserRole": "citizen", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SendingClaimantNotification", "UserRole": "citizen", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SendingRPA", "UserRole": "citizen", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "PinGenerationOperations", "UserRole": "citizen", "CRUD": "CR"},
```

Comment:
I think we will need above three event to be configured for `caseworker-cmc` role as well

Reply:
we are sending authorisation of citizen for sending these events. So dont think we need caseworker-cmc. correct me if I am wrong